getting_started.sh

```diff
@@ -2,10 +2,16 @@
 
 # Install core and extra requirements
 
-pip install -r requirements/requirements-core.txt
-pip install -r requirements/requirements-examples.txt
-pip install -r requirements/requirements-extra.txt
-pip install -r requirements/requirements-docs.txt
+echo -ne 'Building Dependencies... \r'
+pip install -q -r requirements/requirements-core.txt
+echo -ne 'Building Dependencies... #####               (25%)\r'
+pip install -q -r requirements/requirements-examples.txt
+echo -ne 'Building Dependencies... ##########          (50%)\r'
+pip install -q -r requirements/requirements-extra.txt
+echo -ne 'Building Dependencies... ###############     (75%)\r'
+pip install -q -r requirements/requirements-docs.txt
+echo -ne 'Building Dependencies... ####################(100%)\r'
+echo -ne '\n'
```

Comment:
Nice :) How does it fair with debugging a faulty install? (e.g. can't find a given package etc?)

Reply:
Looks good, still spits out the same errors verbosely as before, one annoying thing about bash scripts is even if line 1 fails, it moves onto line 2 but this is independent of my new changes. Might want to try think of a type of `If line 1 fails DONT progress to line 2` type failsafe but call this one for the `icebox`?